pkg/operator/controllers/checker/internetchecker.go

```diff
@@ -57,56 +68,76 @@ func (r *InternetChecker) Check() error {
 		return err
 	}
 
-	var condition status.ConditionType
-	switch r.role {
-	case operator.RoleMaster:
-		condition = arov1alpha1.InternetReachableFromMaster
-	case operator.RoleWorker:
-		condition = arov1alpha1.InternetReachableFromWorker
+	ch := make(chan error)
+	checkCount := 0
```

Comment:
nit: could use len(instance.Spec.InternetChecker.URLs) below and save this variable